sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -56,6 +56,34 @@ ServiceBusClient client = new ServiceBusClient(connectionString);
 
 To see how to authenticate using Azure.Identity, view this [example](#authenticating-with-azureidentity).
 
+### ASP.NET Core
+
+To inject `ServiceBusClient` as a dependency in an ASP.NET Core app, register the client in the `Startup.ConfigureServices` method:
+
+```csharp
```

Comment:
I believe we'll want this content to be a snippet to ensure that we remain up-to-date and compliable.   I If we're doing this for multiple services,  I wonder if it makes sense to have an ASP.NET sample under the repository root `samples` directory and source all of the snippets from there?

Reply:
I think it should be a part of each service's tests/samples as other snippets. We also need to devise a better way to test these extension methods.

Reply:
I agree with Pavel. Having this content in each service's readme will at least ensure that we don't miss the extension method for some packages (this was the case for Event Grid). Curious if different Azure SDK clients have different lifetime management needs. This might be another reason for each service to have its own content.

Reply:
I wasn't suggesting that we move the README section itself,  just the code that we link to as a snippet.   This is currently inline, but we'll need to update so that it uses our snippet system so the code is compiled and validated during nightly runs.   

The question that I was posing was whether the test for that should live in the suite for each individual service or in some central location since there are dependencies for ASP.NET that would need to be included.